krux_cloud_health/cli.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+"""
+CLI tools for accessing Krux Cloud Health Tech
+"""
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import sys
+
+#
+# Third party libraries
+#
+
+import datetime
+
+#
+# Internal libraries
+#
+
+import krux.cli
+from krux.logging import get_logger
+from krux.cli import get_group
+from krux_cloud_health.cloud_health import CloudHealth, NAME, add_cloud_health_cli_arguments, get_cloud_health
+
+
+class Application(krux.cli.Application):
+    def __init__(self, name=NAME):
+
+        # Call to the superclass to bootstrap.
+        super(Application, self).__init__(name=name)
+
+        self.logger = get_logger(name)
+
+        try:
+            self.cloud_health = get_cloud_health(args=self.args, logger=self.logger, stats=self.stats)
+        except ValueError as e:
+            self.logger.warning(e.message)
```

Comment:
Nit: If this is big enough to fail the whole thing, let's write out in `error()`.